Descent3/BOA.h

```diff
@@ -233,10 +233,10 @@ class pq {
 #define BOAF_TOO_SMALL_FOR_ROBOT 0x8000
 #define BOA_TOO_SMALL_FOR_ROBOT(a, b) ((BOA_Array[a][b] & BOAF_TOO_SMALL_FOR_ROBOT) != 0)
 
-typedef struct connect_data {
+struct connect_data {
   int roomnum;
   int portal;
-} connect_data;
+};
```

Comment:
What's the thought on removing the `typedef` of structs to just naming them?

I will admit I come from a C background where `typedef struct blah {...} blah;` is common. I guess I'm just worried if this will break any plain C code.

Reply:
There is no plain C code in D3.

Reply:
Everything compiles in C++, but D3 C++ is more akin to C and uses libc a lot. We strive to make D3 closer to modern C++ practices